app/theme.ts

```diff
@@ -13,82 +14,16 @@ const customTheme = extendTheme({
       secondary: "#db0a40",
       accentGreen: "#32CD32",
       accentGrey: "#787373",
+      accentOrange: "#E9981F",
+      accentYellow: "#FAD50AD4",
```

Comment:
These colors are not in Design system, but they're used in machine cards for machine type labels, so I've added them here to keep all the colors together